.circleci/config.yml

```diff
@@ -37,3 +49,16 @@ jobs:
       - run:
           name: "build lotus-soup"
           command: pushd lotus-soup && go build -tags=testground .
+  trigger-baseline-testplan:
+    executor: golang
+    steps:
+      - setup
+      - run:
+          name: "download testground"
+          command: wget https://gist.github.com/nonsense/5fbf3167cac79945f658771aed32fc44/raw/2e7ced2b1d0ca54033be57299e3fefb840958dfb/testground-3ee0dae -O ~/testground-cli && chmod +x ~/testground-cli
```

Comment:
This is just a temporary binary of `testground` which includes the latest supervisor changes, and can trigger async jobs. Once we have a testground release, and push binaries for linux to Github, I can replace that URL.